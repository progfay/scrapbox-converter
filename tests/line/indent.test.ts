/* global describe it expect */

import { parse } from '@progfay/scrapbox-parser'
import convert from '../../src'
import pretty from 'pretty'

describe('indent', () => {
  it('Simple indent', () => {
    const input = `Indent
${'\t'}level1
${'\t'}${'\t'}level2
${'\t'}${'\t'}${'\t'}level3
${'\t'}level1
${'\t'}${'\t'}${'\t'}level3
${'\t'}${'\t'}level2`
    const html = convert(parse(input), 'projectName')
    expect(pretty(html)).toMatchSnapshot()
  })
})
