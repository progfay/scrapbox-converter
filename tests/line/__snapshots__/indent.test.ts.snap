// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`indent Simple indent 1`] = `
"<article>
  <h1 class=\\"title\\">Indent</h1>
  <section>
    <ul>
      <li>level1</li>
      <ul>
        <li>level2</li>
        <ul>
          <li>level3</li>
        </ul>
      </ul>
      <li>level1</li>
      <ul>
        <ul>
          <li>level3</li>
        </ul>
        <li>level2</li>
      </ul>
    </ul>
  </section>
</article>"
`;
